.eslintignore

```diff
@@ -0,0 +1,3 @@
+example/dist
+dist
```

Comment:
I would like it if we could have the example compile to dist when published to gh-pages and then the actual code published to dist when we run the publish script, do you think you could add this to this PR or should we move this to a different one? I am open to both options.

Reply:
That's a good idea. I created a new PR #80 with these changes. Thanks for pointing this out!